queries.sql

```diff
@@ -8,3 +9,45 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+/* Milestone 2*/
+
+-- Transaction 1 
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+
+-- Transaction 2
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+COMMIT;
+
+-- Transaction 3
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Same here, please verify your changes you can use this query ``SELECT COUNT(*) FROM ANIMALS;` to do that

Reply:
HI @Ismailco, I did it you can see it in the screenshot that I put in the PR. 
![image](https://user-images.githubusercontent.com/45292867/183982547-50be87d0-5c8e-4b8b-81e5-3a532bce67d4.png)
It wasn't a requirement to copy on queries.sql. I've put only like reference is the reason that I didn't put the verify sentences on the file.